### ISSUE TITLE
Support for namespaced attributes?

### ISSUE THREAD

**Issue by FezVrasta** [CoreOffice/XMLCoder#273]
I have this element: `<locRef xlink:href="AT-07-14-01"/>`, how can I read its value? It looks like `@Attribute` doesn't support namespaces?

**Comment by Joannis** [comment]
Yeah, unfortunately it does not. You could provide custom a `CodingKeys` enum. I we'd benefit a lot from supporting Macros
